lib/msal-browser/src/config/Configuration.ts

```diff
@@ -38,14 +38,15 @@ export type BrowserAuthOptions = {
     navigateToLoginRequestUrl?: boolean;
     clientCapabilities?: Array<string>;
     protocolMode?: ProtocolMode;
+    azureAuthOptions?: AzureAuthOptions;
```

Comment:
This name feels a bit too generic...is there something more specific? e.g. `azureCloud`

Reply:
`azureCloudOptions`?

Reply:
Done